src/components/Results/Results.tsx

```diff
@@ -363,7 +363,28 @@ export function Results() {
             />
           </Toolbar>
         </>
-      ) : undefined}
+      ) : (
+        <>
+          <Toolbar>
+            <ToolbarHandle />
+            <ToolbarIcon icon="toolbarSort" title="Sort" />
+            <ToolbarLabel>
+              <Select
+                id="sortOptions"
+                options={sortOptions}
```

Comment:
good catch! i wanna actually check out the repo to do this, so i can ensure my changes actually work and pass prettier and eslint etc, so i’ll get back to you soon :)

Reply:
done in 5f2f9749c64ec442d8df47147f0849d969c33dc4 but i was unable to verify my changes work, since i get a TypeError in simple-encryptor.ts after the chooseMethod screen